distribution/lib/Standard/Table/0.0.0-dev/src/Excel/Excel_Workbook.enso

```diff
@@ -193,3 +193,13 @@ type Excel_Workbook
                 Excel_Reader.prepare_reader_table Report_Warning <| case address of
                     _ : Excel_Range -> ExcelReader.readRange self.workbook address.java_range java_headers skip_rows row_limit
                     _ : Text -> ExcelReader.readRangeByName self.workbook address java_headers skip_rows row_limit
+
+    ## ALIAS worksheet, get
+       GROUP Standard.Base.Input
+       Reads a worksheet from the workbook.
+
+       Arguments:
+       - name: the name of the worksheet to read.
+    sheet : Text | Integer -> Table
+    sheet self name:(Text|Integer) =
+        self.read_section (Excel_Section.Worksheet name 0 Nothing)
```

Comment:
Suggesting an alternative name for this: `get_sheet` 

IMHO it sounds a bit better here, but not strongly attached to it

Reply:
`sheet` is consistent with the Excel VBA API.